README.md

```diff
@@ -21,6 +21,12 @@ To opt out of downloading precompiled assets, set `FFI_BUILD_FROM_SOURCE=1`:
 FFI_BUILD_FROM_SOURCE=1 make clean all
 ```
 
+## Updating CGO Bindings
+
+The CGO bindings are generated using [c-for-go](https://github.com/xlab/c-for-go).
+To generate bindings yourself, install the c-for-go binary, ensure that it's on
+your path, and then run `make cgo-gen`.
```

Comment:
- Should have some notes about the fact that this is checked into source control. 
- Possible improvment,run the generation in CI and make sure the diff is zero?

Reply:
> Possible improvment,run the generation in CI and make sure the diff is zero?

That's also a good idea. Thanks.